src/ResourceManager/Network/ChangeLog.md

```diff
@@ -242,4 +242,5 @@
     - Fixed issue where UseRemoteGateway property was not being populated in the request to the server
 * Get-AzureRmEffectiveNetworkSecurityGroup
     - Add warning if there is no response from GetEffectiveNSG
-* Add Source property to EffectiveRoute 
\ No newline at end of file
+* Add Source property to EffectiveRoute 
+* Add DisableBgpRoutePropagation property in RouteTable.
```

Comment:
@neaggarwMS please move this snippet to the top underneath the `## Current Release` header